pinot-controller/pom.xml

```diff
@@ -92,6 +92,10 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.pinot</groupId>
+      <artifactId>pinot-java-client</artifactId>
+    </dependency>
```

Comment:
Let's not add this dependency, I'd suggest moving `TableNameExtractor` to `pinot-common` instead.